From 5b8fc80e74d7c47e8bfe0dfb4a3b336f057eb2dd Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@freescale.com>
Date: Wed, 27 Jan 2016 11:39:37 +0530
Subject: [PATCH 0323/1383] arm:dts:ls2085a : Add configure-gfladj property to
 USB3 node

Add "configure-gfladj" boolean property to USB3 node. This property
is used to determine whether frame length adjustent is required
or not

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi
index 5bcfaa0..432043e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls2085a.dtsi
@@ -412,6 +412,7 @@
 		reg = <0x0 0x3100000 0x0 0x10000>;
 		interrupts = <0 80 0x4>; /* Level high type */
 		dr_mode = "host";
+		configure-gfladj;
 	};
 
 	usb1: usb3@3110000 {
@@ -419,6 +420,7 @@
 		reg = <0x0 0x3110000 0x0 0x10000>;
 		interrupts = <0 81 0x4>; /* Level high type */
 		dr_mode = "host";
+		configure-gfladj;
 	};
 
 	smmu: iommu@5000000 {
-- 
2.8.1

